packages/aws-cdk/test/api/bootstrap2.test.ts

```diff
@@ -292,6 +307,32 @@ describe('Bootstrapping v2', () => {
       }));
     });
 
+    test.each([
+      // Default case
+      [undefined, 'AWS_DEFAULT_KEY'],
```

Comment:
Mind also adding the following?
```
['AWS_DEFAULT_KEY', 'AWS_DEFAULT_KEY'],
['arn:aws:kms:us-west-2:111122223333:key/1234abcd-12ab-34cd-56ef-1234567890ab', 'arn:aws:kms:us-west-2:111122223333:key/1234abcd-12ab-34cd-56ef-1234567890ab'],
```
For posterity and completeness's sake.

Reply:
👍